joomla/joomla-cms
# [5.0] Update deleted files list in script.php for 5.0.0-alpha2

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) updates the list of files to be deleted on update in file "administrator/components/com_admin/script.php" to recent changes in the 5.0-dev branch in preparation for the upcoming 5.0.0-alpha2 release.

In detail deleted files from following PR's are added:

- PR #39618 all files `*-es5.js`, `*-es5.min.js` and `*-es5.min.js.gz` below the `/media` folder,
and files
'/media/vendor/mediaelement/js/mediaelement-flash-audio-ogg.swf',
'/media/vendor/mediaelement/js/mediaelement-flash-audio.swf',
'/media/vendor/mediaelement/js/mediaelement-flash-video-hls.swf',
'/media/vendor/mediaelement/js/mediaelement-flash-video-mdash.swf',
'/media/vendor/mediaelement/js/mediaelement-flash-video.swf',
- PR #40147 folder '/plugins/task/demotasks' and all files and folders below that folder
- PR #41028 files
'/libraries/vendor/jfcherng/php-diff/src/languages/readme.txt', (from dependency update)
'/plugins/editors-xtd/pagebreak/pagebreak.php', (from upmerged PR #40743 )

### Testing Instructions

Code review.

Or if you want to make a real test, update a 4.4.0-alpha1 or a 5.0.0-alpha1 to the last 5.0 nightly build to get the actual result, and update a 4.4.0-alpha1 or a 5.0.0-alpha1 to the update package built by Drone for this PR to get the expected result.

### Actual result BEFORE applying this Pull Request

The files mentioned above are still present after updating from a 4.4.0-alpha1 or a 5.0.0-alpha1.

### Expected result AFTER applying this Pull Request

The files mentioned above have been deleted after updating from a 4.4.0-alpha1 or a 5.0.0-alpha1.

### Link to documentations
Please select:
